src/Storage.php

```diff
@@ -31,6 +31,7 @@ public function reload(OperationInterface $operation)
         $operation->setState($source['state']);
         $operation->setDisposition($source['disposition']);
         $operation->setBatchId($source['batchId']);
+        $operation->setFamilyId($source['familyId']);
```

Comment:
You'll need to wrap this with an `array_key_exists`, since old operations won't have the familyId field. (That's one bit of trickiness with how flexible Elasticsearch's documents are -- you're never guaranteed to get back all the fields you might expect.)